src/auth/auth.service.ts

```diff
@@ -0,0 +1,38 @@
+import {Injectable} from '@nestjs/common';
+import {InjectRepository} from '@nestjs/typeorm';
+import {Repository} from 'typeorm';
+import {JwtService} from '@nestjs/jwt';
+import {User} from '../user/entities/user.entity';
+
+import * as bcrypt from 'bcrypt';
+
+@Injectable()
+export class AuthService {
+  constructor(
+        @InjectRepository(User) private userRepository: Repository<User>,
+        private readonly jwtService: JwtService,
+  ) {
+  }
+
+  async validateUser(email: string, pass: string): Promise<any> {
+    const user = await this.userRepository.findOne({
+      where: {email},
+    });
+
+    const password = await bcrypt.compare(pass, user.password);
```

Comment:
```suggestion
    const isSamePassword = await bcrypt.compare(pass, user.password);
```
다음과 같이 boolean임을 알 수 있으면 좋을 거 같습니다 🙃